### PR TITLE
Update changelog for CoreDNS

### PR DESCRIPTION
**Pull request by rossf7**
Makes the changelog look nicer and updates CoreDNS to 1.0.4.

@corest I know your change isn't merged yet but wanted to do this before I forgot!

### CODE REVIEW

File: service/version_bundles.go [hunk 1]
```diff
@@ -185,7 +185,7 @@ func newVersionBundles() []versionbundle.Bundle {
 				},
 				{
 					Component:   "CoreDNS",
-					Description: "CoreDNS 1.0.1 replaces kube-dns.",
+					Description: "Version 1.0.4 replaces kube-dns.",
```
Comment (rossf7) [inline service/version_bundles.go]: Makes the changelog look nicer and updates for 1.0.4.

**Comment by rossf7** [comment]
@marians FYI changelog formatting is fixed for CoreDNS. Thanks for raising!

**Comment by marians** [comment]
Nice! Always cool to see a suggestion be picked up that fast. Thanks.

Branch can be deleted, right?

**Comment by rossf7** [comment]
Branch is gone :)

**Pull request merged**